l10n_es_aeat_mod349/models/mod349.py

```diff
@@ -445,6 +445,22 @@ def _selection_operation_key(self):
             allfields=["l10n_es_aeat_349_operation_key"],
         )["l10n_es_aeat_349_operation_key"]["selection"]
 
+    def _process_vat(self, record, errors):
+        partner_vat = record.partner_vat
+        country_code = "".join(c for c in partner_vat[0:2] if not c.isnumeric())
```

Comment:
El código podría venir del país.

```suggestion
        country_code = record.partner_vat[0:2]
        if country_code.isnumeric():
            country_code = record.country_id.code
```

Reply:
Cambiado, gracias!

Reply:
Revisando esto de nuevo, creo que no nos interesa coger el código del `country_id`, por que precisamente queremos escribir el error en caso de que el NIF no tenga código de país.

Reply:
Pero puesto que Odoo admite poner un NIF sin el código de país si tiene el país rellenado, ¿no deberíamos soportarlo nosotros en este módulo y añadir el código de país donde haga falta? Creo que podríamos usar el método de res.partner que hay en `l10n_es_aeat` para obtener esta información normalizada.

Reply:
Buenas Pedro,

Ahora mismo no tenemos la capacidad para realizar este cambio, pero si quieres hacernos un PR con el cambio te lo revisamos y lo agregamos. Muchas gracias

Reply:
Nada, se queda esto en suspenso hasta que podáis entonces.

Reply:
Listo, se han aplicado los cambios solicitados. Gracias.

Reply:
Hola @pedrobaeza,

Al hacer un rebase, se perdió la funcionalidad que se solicitaba aquí. 
He abierto este PR: https://github.com/OCA/l10n-spain/pull/3544
Disculpa las molestias.

Saludos